workshop/content/english/50-java/60-cleanups/_index.md

```diff
@@ -0,0 +1,25 @@
++++
```

Comment:
Please make sure this file is up to date, there should be a whole section on resources which are retained by default

Reply:
The English version has been updated and the Japanese version has been fixed.